src/analyze.proto

```diff
@@ -14,6 +14,16 @@ message AnalyzeApiRequest {
   AnalyzeTemplate analyzeTemplate = 3;
 }
 
+// Recognizer represents a known or custom recognizer
+message Recognizer {
```

Comment:
We have `PatternRecognizer` in the recognizers store proto. Can we merge them somehow? Is there any inheritance in proto files?

Reply:
talked f2f (covid19 style). there is no inheritance in proto, only composition which is an overhead for this.